packages/terra-table/src/Table.jsx

```diff
@@ -478,6 +480,48 @@ function Table(props) {
     }
   }, [tableColumns, onColumnResize]);
 
+  /**
+   *
+   * @param {HTMLElement} element - The element to check if it is a text input
+   * @returns True if the element is a text input.  Otherwise, false.
+   */
+  const isTextInput = (element) => {
```

Comment:
I just didn't move it because it is currently only used in this one location.  We can always move it if the function needs to be shared by multiple files.